### PR TITLE
Add title to admin set edit form

### PR DESCRIPTION
**Pull request by laritakr**
fixes https://github.com/samvera/hyrax/issues/2601

@samvera/hyrax-code-reviewers

### CODE REVIEW

**Comment by laritakr** [comment]
I'm not sure... I copied from the collection edit view which doesn't have one. There is only one spec which covers the <% provide :page_header do %> code in the entire app.

**Comment by elrayle** [comment]
@laritakr Can you test for this using a feature test?  You can add it to features/dashboard/collection_spec.rb and make the code similar to the following updated to one of the admin sets.

https://github.com/samvera/hyrax/blob/master/spec/features/dashboard/collection_spec.rb#L768-L779

**Comment by elrayle** [comment]
@cjcolvar Can you do a final review?  I added a feature test to cover the change in this PR.

There was one random Travis failure.  I've kicked it off again.  I expect it to pass since it did everywhere else.

File: spec/features/dashboard/collection_spec.rb [hunk 1]
```diff
@@ -760,48 +760,70 @@ def get_url_fragment(type)
...
+        end
       end
 
-      it "edits and update collection metadata" do
-        # URL: /dashboard/my/collections
-        expect(page).to have_content(collection.title.first)
-        within("#document_#{collection.id}") do
-          find('button.dropdown-toggle').click
-          click_link('Edit collection')
+      context 'for a collection' do
```
Comment (cjcolvar) [inline spec/features/dashboard/collection_spec.rb]: I'm guessing you mean this to be `admin set`.

**Pull request merged**